### DISCUSSION TITLE
Sorry, you need to login first to access this service.

### DISCUSSION THREAD

**Discussion by codelauncher444** [acheong08/EdgeGPT#295] [Q&A]
The system did not work today, although I renewed the cookies.                                                                                                                 
     File "C:\Users\90533\AppData\Local\Programs\Python\Python311\Lib\site-packages\EdgeGPT.py", line 255, in __init__
    raise NotAllowedToAccess(self.struct["result"]["message"])
EdgeGPT.NotAllowedToAccess: Sorry, you need to login first to access this service.

Is that happening to everyone? If not, can someone help me to solve this issue?

**Comment by KyoshiHikari** [comment]
The indicated error occures, when youre not allowed to use Bing Chat GPT so you need an account that have access to it.

**Reply by DreamMaoMao** [reply]
i have same problem  but i can access to service in web browser  ,it isn't an account problem

**Reply by KyoshiHikari** [reply]
You need to use the cookies obtained from the browser.
1. Obtain cookies
2. Save to a file (ex. cookies.json)

**Reply by DreamMaoMao** [reply]
i found why i receive this message
because i use linux os，and i install this module by pip  but module in linux pip  is not latest version
